client/my-sites/stats/site.jsx

```diff
@@ -613,6 +628,15 @@ export default connect(
 		const upsellModalView =
 			config.isEnabled( 'stats/paid-wpcom-v2' ) && getUpsellModalView( state, siteId );
 
+		const siteCreatedTimeStamp = getSiteOptions( state, siteId ?? 0 )?.created_at;
+
+		// TODO: update the date to the release date when the feature is ready.
+		const redirectToPurchase =
+			config.isEnabled( 'stats/checkout-flows-v2' ) &&
+			isSiteJetpackNotAtomic &&
+			siteCreatedTimeStamp &&
```

Comment:
Hi @grzegorz-cp,

Me and @a8ck3n were talking about this today, and we suspected that the created time might not be the time we are after here. We might be after the 'connected' time, which is the time when the site is connected 🤔 

I'm happy for it to merge and iterate still tho 😄

Reply:
Sorry for pointing probably the wrong one for you.

Reply:
Let's merge it and potentially change it in the next PRs. I'm curious about the scenarios where `connected` makes a difference.